src/category_theory/monoidal/Mod.lean

```diff
@@ -0,0 +1,126 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import category_theory.monoidal.Mon_
+
```

Comment:
None of this file is new, it's just copied and pasted out of `category_theory.monoidal.Mon_` to its own file.